tools/github-bot/src/commands/tools.ts

```diff
@@ -32,6 +31,26 @@ export const commands = (
     const issue = payload.issue;
     const comment = payload.comment;
     const command = comment.body.match(matcher);
+    const login = payload.sender.login;
+
+    const res = await octokit.orgs.checkMembershipForUser({
+      org: "LedgerHQ",
+      username: login,
+    });
+
+    if (res.status > 300) {
```

Comment:
Even though the response can be `302` or `404` (according to the REST API doc https://docs.github.com/en/rest/orgs/members?apiVersion=2022-11-28#check-organization-membership-for-a-user--status-codes ), I believe it would make more sense to use `>=` here: we are checking whether the status is not `2xx`.